spring-cloud-dataflow-docs/src/main/asciidoc/using-spring-cloud-stream-modules.adoc

```diff
@@ -9,6 +9,36 @@ This section dives into the details of using the modules from Spring Cloud Strea
 [[spring-cloud-stream-modules-sources]]
 == Sources
 
+[[spring-cloud-stream-modules-file-source]]
+=== File
+
+The file source provides the contents of a File as a byte array by default. However, this can be
```

Comment:
It is good practice to start a new sentence from a new line for AsciiDoc.
Don't ask me "why?". The advice from someone around here :smile: